lib/export.js

```diff
@@ -2972,7 +2972,11 @@ class FHIRExporter {
       // Allow this for "base" classes
       if (!profile.id.endsWith(`-${profile.type}`)) {
         if (!def._isAbstract) {
-          logger.warn('The \'%s\' property is not bound to a value set, fixed to a code, or fixed to a quantity unit. This property is core to the target resource and usually should be constrained. ERROR_CODE:03006', path);
+          if (def.basedOn[0].name == profile.type) {
+            logger.warn('The \'%s\' property is not bound to a value set, fixed to a code, or fixed to a quantity unit. This property is core to the target resource and usually should be constrained. ERROR_CODE:03006', path);
+          } else {
+            logger.info('The inherited \'%s\' property is not bound to a value set, fixed to a code, or fixed to a quantity unit. This property is core to the target resource and usually should be constrained.', path);
+          }
```

Comment:
I agree this would be a better system. From what I can type, parameterization of error levels is not immediately supported Bunyan; however, I imagine it would be possible through a bit of clever customization.

Reply:
Yeah, I was thinking we'd have our own log function that calls the bunyan logger.  So instead of calling the logger directly, we always call our own log function.  But I haven't fully thought it through, so there may be issues.